lib/src/chart/line_chart/line_chart_data.dart

```diff
@@ -1651,6 +1655,12 @@ class LineTouchTooltipData with EquatableMixin {
   /// Applies a bottom margin for showing tooltip on top of rods.
   final double tooltipMargin;
 
+  /// Controls showing tooltip on left side, right side or center aligned with spot, default is center
+  final FLTooltipAlign tooltipAlign;
+
+  /// Applies horizontal offset for showing tooltip, default is zero.
+  final double tooltipHorizontalOffset;
```

Comment:
You need to add these variables in `List<Object?> get props => []`  (look at the bottom of the class)

Reply:
I missed that part, I'll fix it.